### DISCUSSION TITLE
Disable "Always on Top" for chat window?

### DISCUSSION THREAD

**Discussion by pwnified** [intitni/CopilotForXcode#502] [Ideas]
It's great that the chat window stays on top when docked in Xcode. But if we drag the chat window to another monitor, I want it to have normal window behavior. And go behind other windows. Maybe an option to disable this Always on top behavior? Regards

**Comment by intitni** [comment]
Yes, it's planned. But I am not sure if it's possible to detect if there is no Xcode window on the same screen, so the implementation may be just a toggle to control the always-on-top behavior. I will give it a try in the coming major update.

**Reply by intitni** [reply]
But maybe we can detect if the chat window intersects an Xcode window?

**Comment by intitni** [comment]
Released in 0.33.0 beta 2

<img width="782" alt="Screenshot 2024-05-10 at 15 03 00" src="https://github.com/intitni/CopilotForXcode/assets/793147/34599e03-2859-475d-9b14-a7fd6da1b3a7">

**Reply by pwnified** [reply]
Nice, but one thing. The chat window now goes behind other windows properly, however when clicking on the main Xcode window the chat window suddenly comes forward again. This is with the chat window on another monitor and not overlapped with Xcode. I have the first checkbox on and the second checkbox off.

**Reply by intitni** [reply]
Fixed in beta 3

**Reply by pwnified** [reply]
Works great

**Reply by intitni** [reply]
Released in 0.33.0
